PluginCore/PluginCore/Controls/CompletionList.cs

```diff
@@ -19,37 +19,25 @@ public class CompletionList
         /// <summary>
         /// Properties of the class 
         /// </summary> 
-        private static System.Timers.Timer tempo;
-        private static System.Timers.Timer tempoTip;
-        private static System.Windows.Forms.ListBox completionList;
+        internal static CompletionListControl completionList;
```

Comment:
Do you mean to remove the CompletionList class? It is for making the change completely backwards compatible. Plus removing it would require to change quite several things, but would be possible of course.

There is a comment in the code talking about how it could be done.